docs/tails_printing_guide.rst

```diff
@@ -48,8 +48,8 @@ Tails.
 Connect the printer to your Tails-booted computer via USB, then turn the printer
 on.
 
-Now, you'll want to single-click your way through **Applications** ▸
-**System Tools** ▸ **Settings** ▸ **Devices** ▸ **Printers**. The screenshot
+Now, you'll want to single-click your way through **Applications ▸
+**System Tools ▸ Settings ▸ Device ▸ Printers**. The screenshot
```

Comment:
Nit: The section is still called `Devices` not `Device`

Reply:
fixed and rebased in.